tools/tplg_parser/CMakeLists.txt

```diff
@@ -64,7 +64,7 @@ endif()
 
 # TODO: add IPC4 option when it's ready.
 target_compile_options(sof_tplg_parser PRIVATE
-	-g -O -Wall -Werror -Wl,-EL -fPIC -DPIC
+	-g -O -Wall -Werror -fPIC -DPIC
```

Comment:
Do you remember why you specified the endianness in commit 0c35f1efde17bbd27d?

Reply:
No, I don't. I have probably copied it from somewhere else from that time SOF, and it looks totally unnecessary. Thanks for checking this draft!